is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
+  array.forEach((currentElement, currentIndex) => {
+    array.slice(currentIndex + 1).forEach((elementToCompare) => {
```

Comment:
This syntax is a little confusing -- does it make more sense to compare currentElement to array[currentIndex + 1] instead of using array.slice(...).forEach(...)?